<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>232</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Select The Software</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../civiltools.qrc">
    <normaloff>:/general/images/general/connect.svg</normaloff>:/general/images/general/connect.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QRadioButton" name="etabs_radiobutton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../civiltools.qrc">
          <normaloff>:/civiltools/images/etabs.png</normaloff>:/civiltools/images/etabs.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="sap2000_radiobutton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../civiltools.qrc">
          <normaloff>:/general/images/general/sap2000.png</normaloff>:/general/images/general/sap2000.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="safe_radiobutton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../civiltools.qrc">
          <normaloff>:/general/images/general/safe.jpg</normaloff>:/general/images/general/safe.jpg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="find_button">
       <property name="text">
        <string>Find</string>
       </property>
       <property name="icon">
        <iconset resource="../civiltools.qrc">
         <normaloff>:/general/images/general/find.png</normaloff>:/general/images/general/find.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="connect_button">
       <property name="text">
        <string>Conncet</string>
       </property>
       <property name="icon">
        <iconset resource="../civiltools.qrc">
         <normaloff>:/general/images/general/connect.svg</normaloff>:/general/images/general/connect.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="softwares_layout" native="true"/>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../civiltools.qrc"/>
 </resources>
 <connections/>
</ui>
